docs/components/query.md

```diff
@@ -367,6 +367,14 @@ Flags:
       --query.timeout=2m         Maximum time to process query by query node.
       --query.max-concurrent=20  Maximum number of queries processed
                                  concurrently by query node.
+      --query.lookback-delta=QUERY.LOOKBACK-DELTA
```

Comment:
@krasi-georgiev You need to run `make docs` again, to update this.

Reply:
done